### ISSUE TITLE
Low Battery Status Null Error with Homebridge Beta 1.3.0 beta 58

### ISSUE THREAD

**Issue by cscott365** [chrisjshull/homebridge-nest#367]
**Describe the bug**
[Nest] initing lock (P) "Entryway Lock": deviceId: **removed** structureId: **removed**
[Entryway Lock@Entryway Lock@Status Low Battery] characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics
[2/15/2021, 9:04:42 PM] Received warning for the plugin 'homebridge-nest' from the characteristic 'Status Low Battery': characteristic was supplied illegal value: null! Home App will reject null for Apple defined characteristics

**To Reproduce**
1. Update to latest Homebridge beta with Yale Nest Lock

**Expected behavior**
Understand battery behavior setting

Include with your bug report this version info:
```sh
node --version **14.15.3 (x64)**
homebridge --version **1.3.0 beta 58**
```
I understand this is an issue with the beta version of the latest Homebridge, but wanted to get the issue reported as early as possible. All the other features appear to be working, but I will continue to check those out.

Thanks for a great plugin!

**Comment by adriancable** [comment]
@cscott365 - great spot - thank you for this! There's a bug in my code and the Nest x Yale lock battery status wasn't being reported. Will be fixed in the next version of the plug-in.

**Comment by adriancable** [comment]
Should be fixed in 4.5.2.
